vsintegration/src/FSharp.ProjectSystem.FSharp/Project.fs

```diff
@@ -39,9 +39,15 @@ namespace rec Microsoft.VisualStudio.FSharp.ProjectSystem
     open Microsoft.VisualStudio.Editors
     open Microsoft.VisualStudio.Editors.PropertyPages
     open Microsoft.VisualStudio.PlatformUI
-    
+
     open EnvDTE
 
+    [<assembly:ProvideCodeBase(AssemblyName = "FSharp.Compiler.Private", CodeBase = @"$PackageFolder$\FSharp.Compiler.Private.dll")>]
+    [<assembly:ProvideCodeBase(AssemblyName = "FSharp.Compiler.Server.Shared", CodeBase = @"$PackageFolder$\FSharp.Compiler.Server.Shared.dll")>]
+    [<assembly:ProvideCodeBase(AssemblyName = "FSharp.UIResources", CodeBase = @"$PackageFolder$\FSharp.UIResources.dll")>]
+    [<assembly:ProvideBindingRedirection(AssemblyName = "FSharp.Core", OldVersionLowerBound = "2.0.0.0", OldVersionUpperBound = "4.4.0.0", NewVersion = "4.4.1.0", CodeBase = @"$PackageFolder$\FSharp.Core.dll")>]
```

Comment:
This is great I just don't think the project system is the place for it to live - I'd put it in whichever assembly is the 'primary' assembly (e.g registers F# as a language service etc). Certainly not one that we're hoping to delete soon (as there's a high likelihood we'll forget to move this).